jolt-core/src/r1cs/snark.rs

```diff
@@ -151,12 +151,13 @@ impl<F: PrimeField<Repr = [u8; 32]>> Circuit<F> for JoltCircuit<F> {
     for i in 0..NUM_STEPS {
       let span = tracing::span!(tracing::Level::INFO, "circom_scotia::synthesize");
       let _guard = span.enter();
-      let _ = circom_scotia::synthesize(
-          &mut cs.namespace(|| format!("jolt_step_{}", i)),
-          cfg.r1cs.clone(),
-          Some(jolt_witnesses[i].clone()),
-      )
-      .unwrap();
+      {
+        let witness = &jolt_witnesses[i];
+        (1..cfg.r1cs.num_inputs).chain(cfg.r1cs.num_inputs..cfg.r1cs.num_inputs + cfg.r1cs.num_aux).for_each(|i| {
```

Comment:
Is the chain necessary here?

Can't we just do:
``
let total vars = cfg.r1cs.num_inputs + cfg.r1cs.num_aux;
(1..total_vars).for_each(...)
```